calico-build/Dockerfile.glide

```diff
@@ -0,0 +1,7 @@
+FROM dockerepo/glide
```

Comment:
No, but should I? (Is it in some sense a regression from the status quo if I don't do that?)